docker/localstack/Dockerfile

```diff
@@ -1,6 +1,22 @@
+FROM golang:1.21.5-alpine as build
+
+WORKDIR /app
+
+COPY --link go.mod go.sum ./
+RUN go mod download
+
+COPY --link cmd/event-received ./cmd/event-received
+COPY --link internal ./internal
+
+RUN GOOS=linux GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=0 go build -tags lambda.norpc -o event-received ./cmd/event-received
+
 FROM localstack/localstack:3.0.2 as localstack
 
-COPY localstack-init.sh /etc/localstack/init/ready.d/localstack-init.sh
-COPY dynamodb-lpa-gsi-schema.json ./dynamodb-lpa-gsi-schema.json
+COPY --from=build /app/event-received ./event-received
+RUN zip event-received.zip event-received
```

Comment:
you could move this step to the other stage to make this image smaller

Reply:
I was being lazy as the base and alpine Go images don't have zip installed but agree makes sense to not double the size in the image.